### PR TITLE
Display different background images

### PR DESCRIPTION
**Pull request by bettyMelania**

### CODE REVIEW

**Review by tomagb — APPROVED** [review]

File: frontend/src/Content.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  useEffect(() => {
+    ;(async () => {
+      const bgImages = await getBackgroundImages()
+      setBackgroundImages(bgImages)
+      setLoading(false)
+    })()
+  }, [])
+
+  return loading ? (
+    <div className="spinner-border" role="status" />
```
Comment (campanflaviu) [inline frontend/src/Content.tsx:21]: I know this div was just moved, but maybe you could choose a better naming for className. You should not mention in a class name any styling properties (like border, color, etc). This one could simply be `spinner`

File: frontend/src/api/background.tsx [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
   return axios
     .get(url)
     .then((response) => {
+      const attrs = response.data.data.attributes
       return {
-        homeDesktop: response.data.data.attributes.homeDesktop.data.attributes.url,
-        homeMobile: response.data.data.attributes.homeMobile.data.attributes.url
+        home: attrs.home.data ? attrs.home.data.attributes.url : '',
+        program: attrs.program.data ? attrs.program.data.attributes.url : '',
+        contact: attrs.contact.data ? attrs.contact.data.attributes.url : ''
```
Comment (campanflaviu) [inline frontend/src/api/background.tsx:13]: Optional chaining operator would make these object properties much simpler, like this:
```
...
home: attrs.home?.data.attributes.url || ''
...
```

File: frontend/src/routes/home/Home.tsx [hunk 3]
```diff
@@ -52,47 +55,53 @@ function Home(props) {
...
-          <span className="bold-title">{content.title}</span>
-          <div className="parag">{content.description}</div>
-          <div className="default-red-button">
-            <Link to="/program" onClick={() => setActiveRoute('program')}>
-              AFLĂ PROGRAMUL
-            </Link>
+  const returnContent = () => {
+    return (
+      <>
+        <div className="fullScreen-wrapper">
```
Comment (campanflaviu) [inline frontend/src/routes/home/Home.tsx:61]: just a suggestion for the future, never mix camel-case with dashes in className. As a convention, classNames should always be dash separated. In this example you could choose `fullscreen-wrapper` or `full-scree-wrapper`.

File: frontend/src/routes/home/Home.tsx [hunk 4]
```diff
@@ -52,47 +55,53 @@ function Home(props) {
...
+          <div className="left-title-section">
+            <span className="bold-title">{content.title}</span>
+            <div className="parag">{content.description}</div>
+            <div className="default-red-button">
```
Comment (campanflaviu) [inline frontend/src/routes/home/Home.tsx:65]: Too generic className, and also specifying the color. You could do something like `program-link` or `program-btn`

**Comment by campanflaviu** [comment]
Looks good overall, just a couple of suggestions. I know that most of them are not related to the current PR, but we can do some refactoring as we go.

File: frontend/src/Content.tsx (hunk 1 above)
Comment (bettyMelania) [reply frontend/src/Content.tsx:21]: the className is for bootstrap :)

File: frontend/src/util/constants.ts [hunk 5]
```diff
@@ -12,5 +12,4 @@ export const LINKS = {
   confidential: ''
 }
 
-export const stagingAPI = 'https://adventistclujro-strapi-test.azurewebsites.net/'
-// local export const stagingAPI = 'http://localhost:1337'
+export const host = process.env.REACT_APP_HOST
```
Comment (campanflaviu) [inline frontend/src/util/constants.ts:15]: I think we still need a fallback for the case when APP_HOST is not set, something like this:
```
export const host = process.env.REACT_APP_HOST || 'https://adventistclujro-strapi-test.azurewebsites.net/'
```

**Pull request merged**